### DISCUSSION TITLE
关于数据迁移的问题

### DISCUSSION THREAD

**Discussion by AfterDuang** [YunYouJun/valaxy#182] [Q&A] [answered]
是不是使用文档的内容还在施工啊，有些配置还不知道怎么弄，文章迁移到Valaxy后插入的图片丢失了

**Comment by YunYouJun** [comment, marked as answer]
插入的图片比如？

因为各种边界情况需要反馈才知道会遇到。

**Reply by AfterDuang** [reply]
我的文章迁移之后图片都加载不出来了，还有一些md的代码块错了，这个可能是缩进问题

**Reply by YunYouJun** [reply]
图片加载不出来应该是路径静态资源问题，你可以提供一下你现在的链接，我给你示意一下。

**Reply by AfterDuang** [reply]
解决了，我把Imgae文件夹同步到了Public目录下就行了，以前我都是直接放Source _posts里的

**Reply by YunYouJun** [reply]
嗯，静态资源统一放在 `public` 目录下。
